target/linux/ramips/image/mt7621.mk

```diff
@@ -656,26 +656,51 @@ define Device/wsr-600
 endef
 TARGET_DEVICES += wsr-600
 
-define Device/zbt-we1326
-  DTS := ZBT-WE1326
+define Device/zbt-we1326-256m
```

Comment:
If you are renaming anyway, take the chance to synchronize device name here with compatible in DTS.
Just use `define Device/zbtlink_zbt-we1326-256m` (`_` instead of `,`) and similar for the other devices.
With this, you can get rid of the entries in ramips.sh, too, as those will default to the DTS compatible.